string action_id # MongoDB ObjectId of the action
int32 step_num # Step number to get the state for
---
string prefix # The prefix the client should use to see the robot.
string error # Empty string if no error
